src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFilter.cpp

```diff
@@ -243,10 +250,14 @@ namespace OpenMS
                 c_qc_pass = false;
                 c_qc_fail_message_vec.push_back("metaValue[" + kv.first + "]");
               }
+              ++c_tests_count;
```

Comment:
If the metaValue does not exist or was not found, wouldn't this erroneously count the non existing metaValue towards the overall score?  In general, we should place the iterate in blocks where they will be incremented only if the metaValue or score actually exists.

Reply:
Thanks for pointing this part out. I did not notice that the method was actually implemented within the same class. I thought it was some _MetaInfoInterface_ method.

I'll fix this part.